parser/parser_test.go

```diff
@@ -16,7 +16,7 @@ package parser
 
 import (
 	"bytes"
-	"reflect"
+	"github.com/google/blueprint/proptools"
```

Comment:
Have you made any customizations to Gogland that automate things like line lengths and import ordering? If so, I wonder if we can check them in somewhere.